bot/cogs/snakes.py

```diff
@@ -40,8 +70,93 @@ def __init__(self, bot: AutoShardedBot):
         :param ctx: Context object passed from discord.py
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         """
+        # await ctx.send(BeautifulSoup(text, 'lxml').find("title"))
+        name = str(name)
+        site = 'https://en.wikipedia.org/wiki/' + name
+        async with aiohttp.ClientSession() as session:
+            async with session.get(site) as resp:
+                text = await resp.text()
+                soup = BeautifulSoup(text, 'lxml')
+                title = soup.find('h1').text
+                description = soup.find('table').text
+
+                em = discord.Embed(title=title, description=description)
+
+        if name.lower() == 'python':
+            await ctx.send(await self.get_snek(name))
+        else:
+            await ctx.send(embed=em)
+        # await ctx.send(name)
+
+        # Any additional commands can be placed here. Be creative, but keep it to a reasonable amount!
+    @command()
+    async def snake(self, ctx: Context, x=50, y=30):
```

Comment:
Really fun idea. But maybe a little ambitious for you guys.

Reply:
This is super hard to read, I'm going to have to test it to really see what it does. Maybe you could comment it?

Also, perhaps you could use reactions via `wait_for_reactions()` instead of a list of inputs?